tests/migration_test.go

```diff
@@ -3798,6 +3799,62 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		By("Waiting for VMI to disappear")
 		tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 	})
+
+	Context("with a live-migration in flight", func() {
+		It("there should always be a single active migration per VMI", func() {
+			By("Starting a VMI")
+			vmi := libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			)
+			vmi = runVMIAndExpectLaunch(vmi, 240)
+
+			By("Starting a Migration")
```

Comment:
Why do you create migration before the `Consistently`?

Reply:
to start off with 1 active migration

Reply:
What gives us the guarantee that the migration will not be complete at this point? Should we throttle it?

Reply:
I don't think it's really needed, the test creates 30 + 1 migrations in 30 seconds.

Reply:
I would argue that you then don't need to pre-create the migration but I will not insist.

You don't need to track `createdMigrations` anymore.

Reply:
I can remove it, it's not a big deal.